app/src/main/res/css/modules/column.sass

```diff
@@ -39,13 +39,15 @@
     margin-top: 5rem
     font-size: 1.3rem
   .items_container
-    overflow-x: hidden
-    overflow-y: scroll
-    overflow-x: hidden
+    position: relative
+    overflow: auto
```

Comment:
@tahnik I hear you, but it causes problems with `box-shadow`. If it's going to be `hidden`, there should be enough `padding` for the `box-shadow` IMHO.